composer.json

```diff
@@ -20,7 +20,7 @@
   ],
   "require": {
     "php": "^7.1",
-    "composer-plugin-api": "^1.0.0",
+    "composer-plugin-api": "^1.0 || ^2.0",
     "vlucas/phpdotenv": "^4.0"
```

Comment:
```suggestion
    "vlucas/phpdotenv": "^4.0 || ^5.0"
```

Since the public interface [has not changed](https://github.com/vlucas/phpdotenv/blob/v5.0.0/UPGRADING.md#v4-to-v5) between the two versions.